.github/workflows/windows.yml

```diff
@@ -62,7 +62,8 @@ jobs:
     with:
       github-hosted-runner-label: windows-2022
       self-hosted-image-name: servo-windows10
-      force-github-hosted-runner: ${{ inputs.force-github-hosted-runner }}
+      # FIXME: https://github.com/servo/servo/issues/34603
+      force-github-hosted-runner: true
```

Comment:
@sagudev Once we release the new prebuilt artifacts from mozjs, this can be removed again, since self-hosted runners only fail when building mozjs-sys from source